codalab/lib/fuse.py

```diff
@@ -0,0 +1,978 @@
+# Copyright (c) 2012 Terence Honles <terence@honles.com> (maintainer)
```

Comment:
Hmm.. I copied the file simple because the readme on the github page seems to imply I should just download the file and include it there. Let me see if there's as package published somewhere